com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -286,23 +286,23 @@ public static void NetworkShow(List<NetworkObject> networkObjects, ulong clientI
                     throw new ArgumentNullException("All " + nameof(NetworkObject) + "s must belong to the same " + nameof(NetworkManager));
                 }
             }
-
             using (var buffer = PooledNetworkBuffer.Get())
             using (var writer = PooledNetworkWriter.Get(buffer))
             {
                 writer.WriteUInt16Packed((ushort)networkObjects.Count);
-
+                long prevLength = 0;
```

Comment:
Should this be a ulong?

Reply:
I decided to keep it as a long, as buffer.Length returns a long rather than ulong (see line 300 below), this way I cast it once when we pass it into `TrackObjectSpawnSent`, otherwise I'd have to cast it twice, once on line 300 when we calculate the added length, and once again on line 301.